### DISCUSSION TITLE
C# web page interaction

### DISCUSSION THREAD

**Discussion by MuhammadSamarShehzad** [dotnet/csharplang#6878] [Q&A]
I want to develop a C# console app that interacts with web page user provide it.
i.e; If i have a column on my webpage where the user have to set (tick/cross) sign.
I want that my C# app fill that whole column automatically according to data I provide it.
Kindly recommend me methods for this problem.

**Comment by theunrepentantgeek** [comment]
This project is for discussion of changes to the C# Language itself, it's not a general support forum. 

Suggest you try either the .Net Discord Server or StackOverflow.
